designsafe/apps/api/users/views.py

```diff
@@ -107,7 +107,13 @@ def get(self, request):
         res_list = []
 
         try:
-            users = model.objects.all().filter(username__in=nl)
+            users = []
+            for username in nl:
+                try:
+                    users.append(model.objects.get(username=username))
+                except model.DoesNotExist:
+                    continue
+
```

Comment:
Good catch here! I'm pretty sure this issue with missing user models is only a problem in local/staging environments, but it should prevent some weird bugs from happening during testing/development.